packages/office-ui-fabric-react/src/components/ComponentExamples.test.tsx

```diff
@@ -100,10 +102,15 @@ describe('Component Examples', () => {
   const realToLocaleString = global.Date.prototype.toLocaleString;
   const realToLocaleTimeString = global.Date.prototype.toLocaleTimeString;
   const realToLocaleDateString = global.Date.prototype.toLocaleDateString;
-  const constantDate = new Date(Date.UTC(2017, 13, 6, 4, 41, 20));
+  const constantDate = new Date(Date.UTC(2017, 0, 6, 4, 41, 20));
```

Comment:
seems like an okay fix, but adds to noise to the snapshots in this PR :)

Reply:
13 is not a valid month. I'm not sure how I ever committed it in this state in the first place, but its behavior can vary and needed to be fixed.